mtc.go

```diff
@@ -2039,3 +2044,68 @@ func (tai *TrustAnchorIdentifier) unmarshal(s *cryptobyte.String) error {
 	tai.Issuer = oid
 	return nil
 }
+
+// VerifyOpts includes parameters for verifying a BikeshedCertificate.
+type VerifyOptions struct {
+	// SignedValidityWindow is a signed validity window that covers the
+	// certificate. It is the caller's responsibility to verify the
+	// validity window's signature using the CA parameters.
+	SignedValidityWindow *SignedValidityWindow
+
+	// CA includes the parameters of the CA that issued the batch
+	// containing the certificate.
+	CA *CAParams
```

Comment:
We could instead pass in a `CAStore` and `Lookup` the CA params using `p.TrustAnchorIdentifier().Issuer`.

Reply:
I'm not sure that works because the validity window is specific to a CA. What I think you would want is a callback for looking up the CA parameters and validity window that match the batch id (i.e., issuer id + batch number).

Reply:
Hmm yeah I suppose if we're passing in the ValidityWindow which is already CA-specific, then we might as well pass in the specific CA params as well 👍